### ISSUE TITLE
Does sleep wake-up camera work normally?

### ISSUE THREAD

**Issue by bdragonh** [marco9663/T480s-Hackintosh-Opencore#1]
Does sleep wake-up camera work normally?

**Comment by marco9663** [comment]
Yes, sleep, wake and camera all works. I have actually made some reference to your EFI.

I made some of the important SSDT myself but I do not know why sleep and wake works.

**Comment by marco9663** [comment]
But there is still problem after waking up, the clock speed will speed up to its maximum can cause battery life problem.
I will try to fix it if I have time.

**Comment by bdragonh** [comment]
discover problems

1. Insufficient USB power supply

2. SD card reader does not work

It's better to be based on https://github.com/bdragonh/T480S-EFI The only problem is that Windows does not support it well and crashes after hibernation
